app/controllers/miq_ae_class_controller.rb

```diff
@@ -2014,8 +2013,8 @@ def fields_get_form_vars
         return
       end
       new_fields = {}
-      field_attributes.each_with_object({}) { |field| field = field.to_sym
-      new_fields[field] = @edit[:new_field][field]}
+      field_attributes.each_with_object({}) { |field| field
+      new_fields[field] = @edit[:new_field][field.to_sym]}
```

Comment:
@mkanoor code block end at the end of this line with a curly brace, i can rewrite this block if you want, let me know.